tests/positive/passwd/users.go

```diff
@@ -160,6 +163,172 @@ ENCRYPT_METHOD SHA512
 	}
 }
 
+// DeletePasswdUsers verifies that the user(s) can be removed
+// from a given OS distro.
+func DeletePasswdUsers() types.Test {
```

Comment:
Let's add a case where we specify a user/group shouldn't exist where it doesn't already exist as well.

Reply:
Fixed